projects/fal/src/fal/auth/auth0.py

```diff
@@ -169,6 +169,7 @@ def validate_id_token(token: str):
         algorithms=AUTH0_ALGORITHMS,
         issuer=AUTH0_ISSUER,
         audience=AUTH0_CLIENT_ID,
+        leeway=60,  # 1 minute, to account for clock skew
```

Comment:
This makes it possible for `id_tokens` with an `iat` value **slightly later** than __now__ in the local user's computer to be valid. 

Example:

1. auth0 returns a token with `iat of  `today at 16:22:20`
2. my local computer thinks that right now it is actually `today at 16:22:18` because my clock is not sycned perfectly

This example would normally fail beacuse `16:22:20` is in the future (i.e. `iat < 16:22:18` is NOT true)

Now with the leeway, we leave room for up to a 1 minute error to be there.

So we expect the `iat` to be at bigger than `now - 60 secs`, so we would actually check `iat < 16:23:18` in our example and it would work.

Reply:
This is for `id_token` which is a token intended for the fal client to process and store, no one else.